Kitodo/src/main/webapp/WEB-INF/templates/includes/projects/workflowList.xhtml

```diff
@@ -59,7 +59,7 @@
                             outcome="workflowEdit"
                             styleClass="action"
                             title="#{msgs['view']}"
-                            rendered="#{(item.status ne 'draft') and ((SecurityAccessController.hasAuthorityToViewWorkflow() and not SecurityAccessController.hasAuthorityToEditWorkflow()) or (SecurityAccessController.hasAuthorityToEditWorkflow()))}">
+                            rendered="#{(SecurityAccessController.hasAuthorityToViewWorkflow() and not SecurityAccessController.hasAuthorityToEditWorkflow())}">
                         <h:outputText><i class="fa fa-eye fa-lg"/></h:outputText>
```

Comment:
Is this change correct? Previously users with the authority of editing workflow could view workflow too.